create_auction_result.php

```diff
@@ -1,33 +1,121 @@
 <?php include_once("header.php")?>
+<?php include_once("create_auction.php")?>
+<?php include_once("database.php"); ?>
 
 <div class="container my-5">
 
 <?php
 
-// This function takes the form data and adds the new auction to the database.
+    session_start();
+    global $connection;
+    if (!isset($_SESSION) || $_SESSION == null) {
+    echo('<div class="text-center">You\'re not logged in. Please re-login if this was a mistake</div>');

```

Comment:
I see the "not logged in" logic to be present here - however, this means the page is first rendered, the user fills in the details and then when they submit, they are told that they aren't logged in. It might be beneficial to trigger an alert from the first page's GET request itself and then cause a direct redirection to the log-in page